core/src/engine.cpp

```diff
@@ -24,11 +34,13 @@ Engine::Engine() {
 }
 
 void Engine::update_modules() {
+    double dt = m_windowModule->get_delta_time();
+    currentTime += dt;
```

Comment:
we need to add a helper function to window to manage time passed as it is already a glfw variable we can get

Reply:
done!

Reply:
nice c: